tests/fixtures/incl.h

```diff
@@ -0,0 +1,5 @@
+#define CONST1 42
+#define MACRO(x,y) x+y
+#define MULTI_LINE abc \
+                   xyz
+#define CONST2 99
```

Comment:
there is no newline at the end of the file - diff unhappy.

Reply:
Fixed.